config/routes.rb

```diff
@@ -140,7 +140,7 @@ def external_redirect(path_prefix, target)
     resources :operational_fields, path: 'fields-of-operation', only: [:index, :show]
     get 'world/organisations/:organisation_id/office' => redirect('/world/organisations/%{organisation_id}')
     get 'world/organisations/:organisation_id/about' => redirect('/world/organisations/%{organisation_id}')
-    resources :worldwide_organisations, path: 'world/organisations', only: [:show, :index], localised: true do
```

Comment:
IRL conversation: the API route is defined in another part of `routes.rb`, so this can stay!